Service-Broker-Setup.md

```diff
@@ -0,0 +1,186 @@
+# Setting up and using a service broke sidecar
+
+We currently provide helm charts for two service brokers managing
+access to `MySQL` and `Postgres` databases.
+
+This document describes how to use these charts in the context of
+a CAP cluster.
+
+While the document's examples focus on deploying a `MySQL` database
+the steps are virtually identical for `Postgres`. The
+[Postgres](#appendix-i-postgres) appendix describes the
+differences.
+
+The second appendix lists all relevant chart variables and their
+meanings.
+
+# Assumptions of this document
+
+## About the chart to be deployed
+
+* The user knows how to get the service broker charts, or has them
+  already. This also implies that the user knows the location of the
+  charts in the filesystem.
+
+* The user has stored the path to the chart in the environment
+  variable `CHART`.
+
+  ```
+  CHART=/path/to/mysql-chart
```

Comment:
What happens if it's a published chart?

Reply:
See line 21-23.

Reply:
Right, but in the published chart case it wouldn't be on the filesystem.

Meh, close enough, will merge :D